docs/docs-beta/docs/guides/data-modeling/metadata.md

```diff
@@ -143,13 +156,13 @@ Dagster can automatically attach code references to your assets during local dev
 
 ### Attaching custom code references for local development
 
-You can manually add the `dagster/code_references` metadata to your asset definitions to attach custom code references. This is useful when your asset's source code is not primarily in Python (e.g. when parsing a YAML config or integrating with another language).
+You can manually add the `dagster/code_references` metadata to your asset definitions to attach custom code references. This is useful when your asset's source code isn't primarily in Python (e.g. when parsing a YAML config or integrating with another language).
 
 <CodeExample filePath="guides/data-modeling/metadata/custom-local-references.py" language="python" title="Custom local source code references" />
 
 ### Attaching code references in production (Dagster+)
```

Comment:
Perhaps combine these sections and put `Dagster+` and `OSS` into tabs? Let me know if you'd prefer I do this as a follow up - just throwing it out there since you've been stoked to work on these :)

![Screenshot 2024-08-26 at 1 54 02 PM](https://github.com/user-attachments/assets/5e9b3532-a905-4481-963d-471aa443c570)

Code:

```
### Attaching code references in production

<Tabs>
  <TabItem value="dagster-plus" label="Dagster+">

Dagster+ can automatically annotate your assets with code references to source control such as GitHub or GitLab.

<CodeExample filePath="guides/data-modeling/metadata/plus-references.py" language="python" title="Production source code references (Dagster+)" />

</TabItem>
<TabItem value="dagster-open-source" label="OSS">

If you aren't using Dagster+, you can annotate your assets with code references to source control, but it requires a little more manual mapping.

<CodeExample filePath="guides/data-modeling/metadata/oss-references.py" language="python" title="Production source code references (OSS)" />

[`link_code_references_to_git`](/todo) currently supports GitHub and GitLab repositories. It also supports customization of how file paths are mapped; see the [`AnchorBasedFilePathMapping`](/todo) API docs for more information.

</TabItem>
</Tabs>
```